lib/suse/connect/api.rb

```diff
@@ -230,6 +231,20 @@ def system_migrations(auth, products, target_base_product: nil, kind:)
       def list_installer_updates(product)
         @connection.get('/connect/repositories/installer', params: product.to_params)
       end
+
+      # Search packages which are available in the product of the base product
+      #
+      # @param product [SUSE::Connect::Zypper::Product] the product for in which product tree should be searched
+      # @param query [String] The package query to search
+      #
+      # @return [Array< <Hash>>] of all matched packages available
+      def package_search(product, query)
```

Comment:
Do we want documentation on this?

Reply:
what do you mean by this?

Reply:
I mean whether we want to add a section about this endpoint here: https://github.com/SUSE/connect/blob/master/doc/SCC-API-(Implemented).md. 
I'm not sure as it doesn't say so on the card, but we documented most of the endpoints there. If we decided on that already, it's fine :)

Reply:
I asked @digitaltom about it and we decided to not put it into the documentation for now

Reply:
Ok!